libraries/AP_Scripting/applets/VTOL-quicktune.lua

```diff
@@ -244,12 +281,14 @@ function adjust_gain(pname, value)
          return
       end
       param_changed[iname] = true
-      if string.sub(pname, 1, 3) == "YAW" then
-         -- for yaw, I is 1/10 of P
-         I:set(value*0.1)
-      else
-         -- otherwise I == P
-         I:set(value)
+
+      -- work out ratio of P to I that we want
+      local PI_ratio = QUIK_RP_PI_RATIO:get()
+      if param_axis(pname) == "YAW" then
+         PI_ratio = QUIK_Y_PI_RATIO:get()
+      end
+      if PI_ratio >= 1 then
+         I:set(value/PI_ratio)
       end
```

Comment:
If the ratio is less than 1 should we send a message or set P=I ?

Should we  prevent mistypes of ratios greater than a certain value or just warn?
```suggestion
      else
          I:set(value)
      end
```

Reply:
i've added docs instead, saying it won't change i at all if the ratio is less than 1